content/docs/sidebar.json

```diff
@@ -399,6 +399,37 @@
         "label": "push",
         "slug": "push"
       },
+      {
+        "label": "queue",
+        "slug": "queue",
+        "source": "queue/index.md",
+        "children": [
+          {
+            "label": "queue kill",
+            "slug": "kill"
+          },
+          {
+            "label": "queue logs",
```

Comment:
Oh one more thing: let's use a custom order in here. Probably the same as in the definition of the [`exp queue` index ref](https://dvc-org-cmdref-dvc-queu-iqfvpe.herokuapp.com/doc/command-reference/queue): start, stop, status, logs, remove, kill

Thanks

Reply:
@pmrowla PTAL 🙏